### PR TITLE
New OCTAVE Cheat Sheet

### PR DESCRIPTION
**Pull request by jophab**
**What does your Instant Answer do?**
Shows Quick reference of octave language

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It isreliable and saves a click

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://folk.ntnu.no/joern/itgk/refcard-a4.pdf

**Why did you choose this data source?**
Its top on search results 

**Are there any other alternative (better) data sources?**
Nope.This is better

**What are some example queries that trigger this Instant Answer?**
octave help,octave programming help,octave language help

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Octave programmmers

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No

**Which existing Instant Answers will this one supercede/overlap with?**
None

**Are you having any problems? Do you need our help with anything?**
No

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

-----
IA Page: https://duck.co/ia/view/octave_cheat_sheet

### CODE REVIEW

**Comment by jophab** [comment]
@moollaza How is it??

File: share/goodie/cheat_sheets/json/octave.json [hunk 1]
```diff
@@ -0,0 +1,891 @@
+{
+    "id": "octave_quick_reference",
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/json/octave.json]: I think it should be ```octave_cheat_sheet```

File: share/goodie/cheat_sheets/json/octave.json [hunk 2]
```diff
@@ -0,0 +1,891 @@
...
+                "key": "C-b",
+                "val": "Move back one character"
+            },
+            {
+                "key": "C-f",
+                "val": "Move forward one character"
+            },
+            {
+                "key": "C-a",
+                "val": "Move the the start of the line"
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/json/octave.json]: Repeated ```the``` twice.

File: share/goodie/cheat_sheets/json/octave.json [hunk 3]
```diff
@@ -0,0 +1,891 @@
...
+            {
+                "key": "std (a)",
+                "val": "Standard deviation"
+            },
+            {
+                "key": "var(a)",
+                "val": "Variance"
+            }
+        ],
+        "KillingandYanking": [
```
Comment (codenirvana) [inline share/goodie/cheat_sheets/json/octave.json]: It should be ```Killing and Yanking```

File: share/goodie/cheat_sheets/json/octave.json (hunk 1 above)
Comment (moollaza) [inline share/goodie/cheat_sheets/json/octave.json]: Yes it should all cheat sheets should have `_cheat_sheet` in the ID

File: share/goodie/cheat_sheets/json/octave.json [hunk 4]
```diff
@@ -0,0 +1,891 @@
+{
+    "id": "octave_quick_reference",
+    "name": "OCTAVE",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/octave.json]: Let's sentence case this: `Octave`

File: share/goodie/cheat_sheets/json/octave.json [hunk 5]
```diff
@@ -0,0 +1,891 @@
...
+    "description": "GNU Octave is a high-level interpreted language, primarily intended for numerical computations.",
+    "metadata": {
+        "sourceName": "ntnu",
+        "sourceUrl": "http://folk.ntnu.no/joern/itgk/refcard-a4.pdf"
+    },
+    "template_type": "reference",
+    "section_order": [
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/octave.json]: There's *way* too much content here. It looks like we're mixing keyboard shortcuts with functions/methods etc? Let's stick to just keyboard shortcuts and essentials.

Cheat Sheets are meant to be a quick reference, not an exhaustive guide.

**Comment by jophab** [comment]
@moollaza @codenirvana How  is it now?

File: share/goodie/cheat_sheets/json/octave.json [hunk 6]
```diff
@@ -0,0 +1,273 @@
+{
+    "id": "octave_cheat_sheet",
+    "name": "Octave",
+    "description": "GNU Octave is a high-level interpreted language, primarily intended for numerical computations.",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/octave.json]: Let's remove "GNU Octave" from here: `A high-level interpreted ..."

File: share/goodie/cheat_sheets/json/octave.json [hunk 7]
```diff
@@ -0,0 +1,273 @@
+{
+    "id": "octave_cheat_sheet",
+    "name": "Octave",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/octave.json]: We can change this to GNU Octact

**Comment by moollaza** [comment]
@jophab it's looking *much* better! :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Pull request merged**